src/database/PostgreSQL.ts

```diff
@@ -266,21 +257,17 @@ export class PostgreSQL implements IDatabase {
     });
   }
 
-  saveGame(game: Game): Promise<void> {
+  async saveGame(game: Game): Promise<void> {
     const gameJSON = game.toJSON();
-    this.client.query(
+    return this.client.query(
```

Comment:
With `es6` #4467 and `async` here I would use `await` versus `then` and `catch`. A general rule of thumb I practice is to only use `then` and `catch` in very rare situations. Even then I wouldn't use them.

```suggestion
    try {
        await this.client.query()
        game.lastSaveId++;
     } catch(err) {
         console.error();
     }
```